.changes/store-plugin-rework.md

```diff
@@ -0,0 +1,24 @@
+---
+"store": minor:breaking
+---
+
+**Breaking change**: Renamed `StoreCollection` to `StoreState`
+
+Changes:
+
+- Disallow calling `create_store` when there're still active stores with the same path alive
```

Comment:
Can't we just return the active one then?

Reply:
That is in `store` now

Reply:
yeah, I was talking in terms of breaking changes cause this sounds like another breaking change or a bug fix, debatable tbh.